terasoluna-gfw-parent/pom.xml

```diff
@@ -532,7 +532,7 @@
     <!-- == TERASOLUNA == -->
     <terasoluna.gfw.version>5.6.0-SNAPSHOT</terasoluna.gfw.version>
     <!-- == Spring Boot == -->
-    <org.springframework.boot.version>2.2.0.RELEASE</org.springframework.boot.version>
+    <org.springframework.boot.version>2.2.2.RELEASE</org.springframework.boot.version>
```

Comment:
https://github.com/terasolunaorg/terasoluna-gfw/blob/48b52d4a3cd3be98a242fb1c95fbb726290ef54f/terasoluna-gfw-parent/pom.xml#L578

- [ ] Why not upgrade tomcat version to `9.0.29` ?
https://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/#dependency-versions-coordinates

Reply:
fix by ffb41813da9432eae4ec1f201a9add85bcf86c30